extensions/amp-image-lightbox/0.1/test/integration/test-amp-image-lightbox.js

```diff
@@ -54,7 +54,7 @@ describe
           win = env.win;
         });
 
-        // TODO(wg-ui-and-a11y, #25675) Flaky on sauce labs.
+        // TODO(wg-ui-and-a11y, #25675) Flaky on Firefox / Safari.
```

Comment:
Have you verified both browsers?

Reply:
Not in this PR, but I am going to do a follow up PR to clean up the skips for safari / firefox. Meanwhile, I've changed the comment to mention cross-browser testing.